tests/unit/cmd/hmsl/test_fingerprint.py

```diff
@@ -3,12 +3,63 @@
 import pytest
 from click.testing import CliRunner
 
+from ggshield.cmd.hmsl.decrypt import load_mapping
 from ggshield.cmd.main import cli
 from ggshield.hmsl.client import PREFIX_LENGTH
 from ggshield.hmsl.crypto import hash_string
 from tests.unit.conftest import assert_invoke_exited_with, assert_invoke_ok
 
 
+ENV_FILE = r"""
+FOO=bar
+PASSWORD=1234
+# Don't take "export" into account
+export SECRET=foo
+
+this line does not contain a secret
+
+# Commented secrets should be detected
+# BAR=prod123
+# But comments should be ignored
+BAZ=spam # eggs
+# All at once
+   # FOOBAR=toto42 # a super secret
+
+# Quotes should be ignored
+PASSWORD2="123v4@!,asTd"
+export PASSWORD3="P@ssw0rd3"
```

Comment:
To fix the broken functional tests you need to ggignore these secrets:

```suggestion
PASSWORD2="123v4@!,asTd"  # ggignore
export PASSWORD3="P@ssw0rd3"  # ggignore
```